tsconfig.base.json

```diff
@@ -17,6 +17,7 @@
     "noImplicitReturns": true,
     "noUnusedLocals": true,
     "noUnusedParameters": false,
+    "esModuleInterop": true,
```

Comment:
If this is relevant only for a single package, let it be defined in package-specific tsconfig only.